react/components/ListDetails/ItemDetails.tsx

```diff
@@ -28,9 +50,12 @@ class ItemDetails extends Component<ItemDetailsProps, ItemDetailsState> {
   }
 
   public render(): ReactNode {
-    const { item: { product } } = this.props
-    const { isSelected, isLoading } = this.state
+    const { item: { product }, lists, intl } = this.props
+    const { isSelected, isLoading, isCopying } = this.state
     const deleteIcon = <IconDelete />
+    const className = classNames(`${wishlist.summaryContainer} h4 bb b--muted-4`, {
```

Comment:
nit: no need for string templates when using `classNames`
```suggestion
    const className = classNames(wishlist.summaryContainer, 'h4 bb b--muted-4', {
```